config/routes.rb

```diff
@@ -1,3 +1,14 @@
 Rails.application.routes.draw do
+  resources :showtimes do
+    get '/', to: 'orders#new'
+    get '/orders', to: 'orders#new'
+
+    resources :orders, only: %i[new create]
+
+    root 'orders#new'
```

Comment:
I feel like there's a better way to handle all of the redirecting to `orders#new`